### PR TITLE
feat(backend): show wallet balance

### PR DESCRIPTION
**Pull request by alex1307**
Description

We should show available and full wallet balance.
Wallet gRpc server alreay exposes such data [get_balance] and we will forward to the Launchpad front-end.

Motivation and Context
Full and available balance should be visible to users.

How Has This Been Tested?
The chagne is tested manually.

### CODE REVIEW

File: applications/launchpad/backend/src/api/wallet_api.rs [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+use tari_app_grpc::tari_rpc::wallet_client;
+use tauri::{http::status, AppHandle, Manager, Wry};
+
+use crate::{
+    commands::{status, AppState, DEFAULT_IMAGES},
+    docker::{ContainerState, ImageType, TariNetwork},
+    grpc::{GrpcWalletClient, WalletTransaction, WalletIdentity, WalletBalance},
+};
+
+
```
Comment (tarnas14) [inline applications/launchpad/backend/src/api/wallet_api.rs]: too much whitespace

Comment (CjS77) [reply applications/launchpad/backend/src/api/wallet_api.rs]: `cargo fmt --all` will sort this out

File: applications/launchpad/backend/src/main.rs [hunk 2]
```diff
@@ -142,6 +141,28 @@ async fn subscribe() {
...
+                Err(err) => error!("Failed to read identity: {}", err),
+            };
+            
+        } else {
+            error!("Wallet is down");
+            continue
+        }
+    }
+}
+
```
Comment (CjS77) [inline applications/launchpad/backend/src/main.rs]: Is this function written as intended? An infinite loop?

**Comment by tarnas14** [comment]
#258

**Review by CjS77 — APPROVED** [review]
Looks like there could be some clippy issues (unused imports). Otherwise lgtm

**Pull request merged**